Mention linux updates (#1984)
privacyguides/privacyguides.org: docs/os/linux-overview.en.md

We **highly recommend** that you install the microcode updates, as your CPU is already running the proprietary microcode from the factory. Fedora and openSUSE both have the microcode updates applied by default.

### Updates

> Most Linux distributions, will update automatically or remind you to so. It is important to keep your OS up to date so that you your software is patched when a software vulnerability is found.

Reviewer: ```suggestion
Most Linux distributions will update automatically or remind you to so. It is important to keep your OS up to date so that your software is patched when a vulnerability is found.
```

> Some distributions (particularly those aimed at advanced users) are more cut down and expect you to do things yourself (e.g. Archlinux or Debian), will require running the "package manager" (`apt`, `pacman`, `dnf` etc) in order to receive important security updates.

Reviewer: ```suggestion
Some distributions (particularly those aimed at advanced users) are more barebones and expect you to do things yourself (e.g. Archlinux or Debian). These will require running the "package manager" (`apt`, `pacman`, `dnf` etc) in order to receive important security updates.
```

> Some distributions will also not download firmware updates automatically. For that you would need to install [`fwupd`](https://wiki.archlinux.org/title/Fwupd).

Reviewer: ```suggestion
Some distributions will also not download firmware updates automatically. For that you will need to install [`fwupd`](https://wiki.archlinux.org/title/Fwupd).
```
